guides/source/active_record_multiple_databases.md

```diff
@@ -378,10 +378,11 @@ like `connected_to(role: :nonexistent)` you will get an error that says
 
 If you want Rails to ensure any queries performed are read only, pass `prevent_writes: true`.
 This just prevents queries that look like writes from being sent to the database.
+This is always enabled for `role: :reading`, but can be set to prevent write queries against any connection.
```

Comment:
This is not technically true if we read documentation above. We can avoid saying this, to push to use read only role for better prevention?

Reply:
> This is not technically true (ie. "any connection")if we read documentation above.

How about `can be set to prevent write queries against any role.`?  

> We can avoid saying this, to push to use read only role for better prevention?

I don't understand.  Are you advocating not documenting this feature?  There are reasons to use this in spite of read only role preventing writes far more effectively.  Perhaps a user must use the `:primary` database connection to avoid replication delay but would like the extra safety of preventing write queries from running against it.

Reply:
@vipulnsward what do you think of the new phrasing?

Reply:
I'm weary about telling people to see `READ_QUERY` something that is intentionally private, afaict. And since we can't actually link to it, just giving people a word to go grep for is questionable. I'd almost prefer we give them an arbitrary list of _potentially but not officially_ supported queries.

Reply:
@zzak I've removed the line about the `READ_QUERY`.  Since each database has different queries which are supported, I've not added a list of examples.